pkg/cmd/datamgr/cli/install/install.go

```diff
@@ -137,14 +143,21 @@ func NewCommand(f client.Factory) *cobra.Command {
 }
 
 func (o *InstallOptions) Run(c *cobra.Command, f client.Factory) error {
-	var resources *unstructured.UnstructuredList
-
-	// In case of Guest or Supervisor cluster, skip installing data manager
 	skipDataMgr := false
-	clusterFlavor, err := utils.GetClusterFlavor(nil)
-	if clusterFlavor == utils.TkgGuest || clusterFlavor == utils.Supervisor {
-		fmt.Printf("The Plugin is installed in %s\n. Skipping data manager installation.", clusterFlavor)
-		skipDataMgr = true
+	installCRDs := true
+	// Check if ItemActionPlugin feature is enabled.
+	if cmd.IsFeatureEnabled(o.Features, utils.VSphereItemActionPluginFlag) {
+		fmt.Printf("Feature %s enabled. Install data manager Only on vanilla", utils.VSphereItemActionPluginFlag)
+
+		// In case of Guest or Supervisor cluster, skip installing data manager
+		clusterFlavor, _ := utils.GetClusterFlavor(nil)
+		if clusterFlavor == utils.TkgGuest || clusterFlavor == utils.Supervisor {
+			fmt.Printf("The Cluster Flavor: %s\n. Skipping data manager installation.", clusterFlavor)
+			skipDataMgr = true
+		}
+
+		// The CRDs are installed as a part of Backup driver and not the data manager
+		installCRDs = false
```

Comment:
What data manager image do you plan to use in the data manager OVF? If we don't install CRDs with the feature flag being enabled, then we cannot use data manager image built from master branch since this commit, as the feature flag is explicitly enabled in the script/install.sh.

Reply:
Not sure I understand. The data manager in the ovf runs as a docker container, and watches for the upload and download CRDs in the kubernetes cluster, where the Backup driver and the CRDs are installed. Will that not be enough?

Reply:
If Backup driver has already installed CRDs, that is enough for data manager for the current design.

But that will set unnecessary dependency on backup driver. Instead, we can always try to install CRDs when installing data manager. The fact is, if CRDs have been installed by Backup driver, it will just be a NoOp to K8s API server.